extensions/interactions/LogicProof/LogicProof.html

```diff
@@ -12,14 +12,14 @@
 
 <script type="text/ng-template" id="interaction/LogicProof">
   <div>
-  <span translate="<[questionString]>" translate-values="<[questionStrData]>"></span>
+  <span translate="<[questionString]>" translate-values="<[questionStringData]>"></span>
     <div class="logic-proof-input-box">
       <div ui-codemirror="{onLoad: codeEditor}">
         <textarea ng-trim="false" rows="10"></textarea>
       </div>
     </div>
-    <button class="btn btn-default" ng-click="submitProof()" translate="I18N_INTERACTIONS_PROOFS_SUBMIT"></button>
-    <button class="btn btn-default" ng-click="showHelp()" translate="I18N_INTERACTIONS_PROOFS_POSSIBLE_LINES"></button>
+    <button class="btn btn-default" ng-click="submitProof()" translate="I18N_INTERACTIONS_LOGIC_PROOFS_SUBMIT"></button>
```

Comment:
These and the rest: suggest LOGIC_PROOF instead of LOGIC_PROOFS, since there's only one proof here (and LogicProof is the interaction id).